### ISSUE TITLE
error installing

### ISSUE THREAD

**Issue by jdmontenegro** [federicomarini/GeneTonic#44]
Hi, I am trying to use GenTonic in our local server, however I am getting the following error which I cannot find a solution for:
```
> BiocManager::install("GeneTonic")
'getOption("repos")' replaces Bioconductor standard repositories, see '?repositories' for details

replacement repositories:
    CRAN: https://cran.rstudio.com/

Bioconductor version 3.15 (BiocManager 1.30.18), R 4.2.0 (2022-04-22)
Installing package(s) 'GeneTonic'
trying URL 'https://bioconductor.org/packages/3.15/bioc/src/contrib/GeneTonic_2.0.2.tar.gz'
Content type 'application/x-gzip' length 9023925 bytes (8.6 MB)
==================================================
downloaded 8.6 MB

* installing *source* package ‘GeneTonic’ ...
** using staged installation
** R
** data
** inst
** byte-compile and prepare package for lazy loading
Error: object ‘universal’ is not exported by 'namespace:backbone'
Execution halted
ERROR: lazy loading failed for package ‘GeneTonic’
* removing ‘/home/user/jmontenegro/R/x86_64-pc-linux-gnu-library/4.2/GeneTonic’
```
The problem seems to be the object "universal" is not exported, Any suggestions to fix this?

Best regards, 

Juan D.

**Comment by federicomarini** [comment]
Hi Juan,
I am aware of the issue, and this is currently fixed on the development version available on GitHub - this is due to the recent changes in the `backbone` package we are importing.
You can install the latest release version, yet you need to do it via `remotes` from GitHub, because the Bioconductor release cycle is getting close to the new released version and commits are frozen to that branch.

Long story short:

```
library("remotes")
remotes::install_github("federicomarini/GeneTonic@RELEASE_3_15")
```

You can make sure it is the right one if you have version 2.0.3 installed.
Federico

**Comment by federicomarini** [comment]
closing this one, should be resolved after the commit
